ext/exif/exif.c

```diff
@@ -3810,12 +3805,6 @@ static bool exif_scan_JPEG_header(image_info_type *ImageInfo)
 
 		fpos = php_stream_tell(ImageInfo->infile);
 
-		if (marker == 0xff) {
```

Comment:
could it be still somewhat useful (e.g. fuzzing) ?

Reply:
I don't understand your question. This check can never be hit because of the do-while loop: if we get here marker must be !=0xff. Unless this code was meant to check in a different way?

Reply:
nevermind I misread the code it s clearer when unfolded.

Reply:
It *might* be a good idea to replace that with a `ZEND_ASSERT()`, so it's easier to understand (and to also have a safeguard just in case the code above would be modified).